rotkehlchen/db/dbhandler.py

```diff
@@ -1293,9 +1297,10 @@ def get_blockchain_account_data(
         Each account entry contains address and potentially label and tags
         """
         query = cursor.execute(
-            'SELECT A.account, A.label, group_concat(B.tag_name,",") '
+            'SELECT A.account, C.name, group_concat(B.tag_name,",") '
             'FROM blockchain_accounts AS A '
             'LEFT OUTER JOIN tag_mappings AS B ON B.object_reference = A.blockchain || A.account '
+            'LEFT OUTER JOIN address_book AS C ON C.address = A.account AND C.blockchain = A.blockchain '  # noqa: E501
```

Comment:
blockchain can be NULL in the schema so compare it using the `IS` operator. Using `=` with nullable columns can lead to errors hard to debug later